src/pages/Reviews.js

```diff
@@ -63,21 +62,24 @@ const Reviews = ({ productId }) => {
         rating: '',
         title: '',
         comment: '',
-        product: productId,
-        user: user,
+        product: productId
       });
-   
+
       setMessage('Review created successfully');
-      getAllReviews(productId);
+      await getAllReviews(productId);
     } catch (error) {
       console.error(error);
-      setMessage('Check all the fields');
+      if (error?.response?.status >= 400 && error?.response?.status < 500) {
```

Comment:
Optional - but I like to just use the server's error message for anything with a status code in the 400s. Because usually any 4xx status code, is meant to indicate an error that the _user_ caused/can fix ( like 400 due to user not inputting all the fields in the form, or 403 because user is not logged in, etc). 

500 errors are usually meant for when the server has an error, in which case, we might not want to show the user the error messagee since it could be very technical. In that case, we could just show a generic "Try again later" message (or like I'm doing here, just fall back to the "Check all the fields" message).